tzrec/modules/backbone_module.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) 2025, Alibaba Group;
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#    http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import torch
+import torch.nn as nn
+
+
+class Add(nn.Module):
+    """Element-wise addition module for multiple tensors.
+
+    This module performs element-wise addition of multiple input tensors.
+    """
+
+    def __init__(self) -> None:
+        super().__init__()
+
+    def forward(self, input1, input2, input3=None):
```

Comment:
Why do we need an Add module? Wouldn't a Lambda module offer more flexibility? Additionally, should the Add module be designed to support inputs like input4 and input5?